### PR TITLE
DOC: Update documentation

### PR DESCRIPTION
**Pull request by ifranzki**

### CODE REVIEW

File: doc/README.icsf_stdll [hunk 1]
```diff
@@ -20,18 +20,19 @@ centralized storage and key management.
...
 The remote server's configuration is outside the scope of this README.
 
-Please see XXXXX for more information on the z/OS LDAP server requirements and
-remote crypto configuration.
-For more information on ICSF PKCS#11, see "Writing PKCS#11 Applications" manual
-at
-http://www-01.ibm.com/support/docview.wss?uid=isg24664157d6cdc890485257a7c0054dc3d&aid=1.
+Please see the related z/OS documentation for more information on the z/OS
+LDAP server requirements and remote crypto configuration, as well as on
+on ICSF PKCS#11.
```
Comment (jschmidb) [inline doc/README.icsf_stdll:25]: two times 'on'

File: doc/README.token_data [hunk 2]
```diff
@@ -6,6 +6,12 @@ in its own file and given a unique name that is never reused. Each object is
 stored in a binary format that gives openCryptoki the ability to check for
 corruption when reading in the object into memory.
 
+Starting with openCryptoki vesion 3.12, a new, FIPS compliant format can be used
+to store the token objects. You may now want to migrate the data stores of a
```
Comment (jschmidb) [inline doc/README.token_data:10]: an EP11 token

File: doc/system_resources [hunk 3]
```diff
@@ -1,27 +1,45 @@
-The following are the system resources used by openCryptoki as of
-openCryptoki-3.8 release.
+The following are the system resources used by openCryptoki
 
-1.Shared memory = 1 per token + 1 segment between pkcsslotd & api = 7 max currently
+Please also see https://www.ibm.com/docs/en/linux-on-systems?topic=features-architecture-components-opencryptoki
+
+1.Shared memory = 1 per token + 1 segment between pkcsslotd & api + 1 statistic
+  segemnt per user (if statistics are enabled)
```
Comment (jschmidb) [inline doc/system_resources:6]: segment

File: doc/README.token_data (hunk 2 above)
Comment (ifranzki) [reply doc/README.token_data]: fixed

File: doc/system_resources (hunk 3 above)
Comment (ifranzki) [reply doc/system_resources]: fixed

**Review by jschmidb — APPROVED** [review]
lgtm

**Pull request merged**